### PR TITLE
Add a "Technical Best Practices" section

### PR DESCRIPTION
**Pull request by andrewchilds**

### CODE REVIEW

File: meetings.md [hunk 1]
```diff
@@ -56,3 +56,15 @@ If there's no need to use a conference room, consider just joining the meeting f
...
 
 ![Zoom video call](/images/zoom-video-call.jpg)
+
+### Technical Best Practices
+
+Whichever software you use to have remote meetings, a few basic things can ensure you and your colleagues are able to speak and be heard clearly.
+
+- **Use headphones, ideally with a noise-cancelling microphone.** Without headphones, you run the risk of creating an echo any time your wifi starts to lag. A noise-cancelling microphone also makes an enormous difference if you’re sitting in the office, coworking space, coffee shop, or any other noisy environment. We’ve had a good experience with the Jabra UC VOICE 550 and the more recent Jabra Evolve 40. It’s a good practice to double-check that Zoom/Meet/etc is actually using your headset as input when you join a call.
+
+- **Plug in whenever possible.** Plugging in to AC power and to ethernet are both known to improve call quality.
```
Comment (victorwuky) [inline meetings.md]: Really?! This is amazing. (I don't have ethernet but I do have power 😄 )

File: meetings.md [hunk 2]
```diff
@@ -56,3 +56,15 @@ If there's no need to use a conference room, consider just joining the meeting f
...
+
+- **Mute until you’re ready to talk.** This can also act as a non-verbal cue that you would like to say something next.
```
Comment (victorwuky) [inline meetings.md]: I love this one. I've seen this _all the time_.

"Victor, did you want to say something, I saw you unmute yourself".

File: meetings.md [hunk 3]
```diff
@@ -56,3 +56,15 @@ If there's no need to use a conference room, consider just joining the meeting f
...
+
+- **Use “Do Not Disturb” mode.** This prevents random Slack/etc alerts from distracting you (and others) during your presentation. (In OS X, click the top right notifications icon in the top right, then scroll up to find the Do Not Disturb toggle, and then click it to enable or disable.)
```
[the remark this answers is not in the document]
Comment (victorwuky) [reply meetings.md]: I think you'll agree anyways. Committing. I'll stop wasting your time with my blabber.

File: meetings.md (hunk 1 above)
Comment (andrewchilds) [reply meetings.md]: The makers of [Tuple](https://tuple.app), a new screensharing product specifically for pair programming, had this to say in an email about improving call quality:

> MacBooks sometimes throttle CPU usage when running on battery power. We're judicious about our CPU utilization, but doing a call while on battery can affect your latency.
> 
> Additionally, a wired Ethernet connection will tend to have lower ping, less packet loss, and fewer ping spikes.
>
> Here's [a great article](https://na.leagueoflegends.com/en/page/ethernet-vs-wifi-ping-packets-playing-better) with data to back that up.
>
> Try to plug in whenever you're about to pair.

**Pull request merged**